client/src/main/java/org/apache/falcon/client/FalconClient.java

```diff
@@ -358,7 +358,9 @@ public void setDebugMode(boolean debugMode) {
         UNREGISTER("api/extension/unregister/", HttpMethod.POST, MediaType.TEXT_PLAIN),
         DETAIL("api/extension/detail/", HttpMethod.GET, MediaType.APPLICATION_JSON),
         JOB_DETAILS("api/extension/extensionJobDetails/", HttpMethod.GET, MediaType.APPLICATION_JSON),
-        REGISTER("api/extension/register/", HttpMethod.POST, MediaType.TEXT_PLAIN);
+        REGISTER("api/extension/register/", HttpMethod.POST, MediaType.TEXT_PLAIN),
+        ENABLE("api/extension/enable", HttpMethod.POST, MediaType.TEXT_PLAIN),
```

Comment:
Guess this will also be TEXT_XML since we are returning APIResult. Sorry, I misled you earlier.